plugins/table.md

```diff
@@ -80,6 +80,28 @@ tinymce.init({
 });
 ```
 
+### `table_use_colgroups`
+
+{{ site.requires_5_5v }}
+
+This option allows you to set whether `colgroup`s should be added to new tables created by the table plugin. Tables inserted by other means, or already existing tables, will not be changed.
```

Comment:
I'd like to take the approach of talking about this like "allows you to use colgroups/cols for columns widths" because technically there's other things you can do with colgroups/cols in HTML but YMMV with TinyMCE (e.g. browsers seem to not support the `span` attr on `col` elements very well, but some styling stuff may work) especially since the only thing that works via our UI is column widths. 

Also, we may tag this as experimental. Need to check with stakeholders.

Reply:
after a quick discussion, we have decided not to mark this as experimental and we have documented the limitations of the feature.